feeds/us-ca.json

```diff
@@ -90,6 +90,30 @@
             "type": "transitland-atlas",
             "transitland-atlas-id": "f-9q5-metro~losangeles~rail"
         },
+        {
+            "name": "Antelope-Valley-Transit-Authority",
+            "type": "transitland-atlas",
+            "transitland-atlas-id": "f-antelope~valley~transit~authority"
+        },
+        {
+            "name": "Glendale-Beeline",
+            "type": "transitland-atlas",
+            "transitland-atlas-id": "f-9q5f-glendalebeeline"
+        },
+        {
+            "name": "Burbank-Bus",
```

Comment:
This one seems to stumble on agency.txt (something with the language code).
Maybe adding:

"fix": true
would resolve this (I'm not entirely sure)